cli/ioctl/cmd/config/configsetget.go

```diff
@@ -110,6 +135,8 @@ func set(args []string) (string, error) {
 		ReadConfig.SecureConnect = !Insecure
 	case "wallet":
 		ReadConfig.Wallet = args[1]
+	case "currentcontext":
+		ReadConfig.CurrentContext = args[1]
```

Comment:
When setting the address, let's try to do some validation. Check ioctl/validator.go